tests/foreman/ui/test_rhc.py

```diff
@@ -65,10 +66,32 @@ def fixture_setup_rhc_satellite(request, module_target_sat, module_rhc_org):
         module_target_sat.cli.Subscription.upload(
             {'file': manifests_path, 'organization-id': module_rhc_org.id}
         )
+        rh_repo1 = {
+            'name': constants.REPOS['rhel8_aps']['name'],
+            'product': constants.PRDS['rhel8'],
+            'reposet': constants.REPOSET['rhel8_aps'],
+            'basearch': constants.DEFAULT_ARCHITECTURE,
+            'releasever': '8',
+        }
```

Comment:
You can use directly `constants.REPOS['rhel8_aps']` instead of `rh_repo1` (you are constructing the same dict structure which is already present in codebase)  

1 caveat: you have to add arch under 'rhel8_aps' - it's missing while for rhel7 it's there, yet you fix this inconsistence

Reply:
nice catch! Updated the pr.